recipes/redis-plus-plus/all/conanfile.py

```diff
@@ -67,7 +67,10 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("hiredis/1.0.2")
+        if Version(self.version) < "1.3.0":
+            self.requires("hiredis/1.0.2")
```

Comment:
hideris 1.1.x is not compatible with redis-plus-plus < 1.3?

Reply:
@SpaceIm 
Thank you for your comments!
I recheck the compatibility of hiredis, and there is no issue about it.
I try to fix this.